lib/src/clojure_lsp/refactor/transform.clj

```diff
@@ -379,17 +382,152 @@
                                        (z/append-child fn-sym)
                                        (z/append-child used-syms)
                                        (z/append-child* (n/newlines 1))
-                                       (z/append-child* (n/spaces 2))
+                                       (z/append-child* (n/spaces (+ defn-col 1)))
                                        (z/append-child expr-node)
                                        z/up)]
           [{:loc   defn-edit
-            :range {:row     (or prev-end-row-w-space form-row)
-                    :col     form-col
-                    :end-row (or prev-end-row-w-space form-row)
-                    :end-col form-col}}
-           {:loc expr-edit
+            :range defn-range}
+           {:loc   expr-edit
             :range expr-meta}])))))
 
+(defn ^:private replace-sexprs [zloc replacements]
+  (z/prewalk zloc z/sexpr-able?
+             (fn [zloc]
+               (when-let [replacement (get replacements (z/sexpr zloc))]
+                 (z/replace zloc replacement)))))
+
+(defn ^:private convert-literal-to-fn-params [zloc]
+  ;; only function literals #(,,,)
+  (when-let [literal-node (z/find-tag zloc z/up :fn)]
+    [literal-node]))
+
+(defn can-convert-literal-to-fn? [zloc]
```

Comment:
could this be private?

Reply:
Yes, good idea... Initially I was thinking `convert-fn-to-literal` and `convert-literal-to-fn` would both be public commands, and would need their own public `can-*?` helpers. But I think it's better the way it is—one public function that does what you want. I'll make these private.